create_student.rb

```diff
@@ -1,5 +1,7 @@
 require_relative 'student'
+require_relative 'person'
```

Comment:
- [ ] It would be better if we are able to run the app without getting errors, in the `person.rb` file, kindly change `require_relative 'decorate_pattern\nameable'` to `require_relative 'decorate_pattern/nameable'`. Otherwise here is the error I'm getting while running the `ruby main.rb` command. Kindly check the screenshot below
![Screenshot from 2022-08-12 06-11-07](https://user-images.githubusercontent.com/46388113/184295204-75984154-3509-402e-9756-0700dd06268b.png)